README.md

```diff
@@ -1,2 +1,149 @@
-# Demos
-Notebooks for ThirdAI demos
+<div id="top"></div>
+
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![License][license-shield]][license-url]
+
+
+
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  <a href="https://github.com/ThirdAILabs/Demos">
+    <img src="https://www.thirdai.com/wp-content/uploads/2021/11/logo-illustrator1.png" alt="Logo" width="150" height="">
+  </a>
+
+<h1 align="center">Demos</h1>
+
+  <p align="center">
+    Interactive notebooks for exploring ThirdAI's library.
+    <br>
+    <br>
+    <a href="https://thirdai.com">[Website]</a>
+    ·
+    <a href="https://thirdai.com/docs/">[Docs]</a>
+    ·
+    <a href="https://github.com/ThirdAILabs/Demos/issues">[Report Bug]</a>
+    ·
+    <a href="https://www.thirdai.com/careers/">[We're Hiring]</a>
+  </p>
+</div>
+
+
+
+<!-- TABLE OF CONTENTS -->
+<br>
+Table of Contents
+<ol>
+  <li>
+    <a href="#👋-welcome">Welcome</a>
+  </li>
+  <li>
+    <a href="#🚀-quickstart">Quickstart</a>
+    <ul>
+      <li><a href="#🪪-downloading-a-license">Downloading a License</a></li>
+      <li><a href="#💾-installation">Installation</a></li>
+    </ul>
+  </li>
+  <li><a href="#🎮-usage">Usage</a></li>
+  <li><a href="#📄-license">License</a></li>
+  <li><a href="#🎙-contact">Contact</a></li>
+</ol>
+
+<br>
+
+
+
+<!-- ABOUT THE PROJECT -->
+# 👋 Welcome
+
+ThirdAI's library is a deep-learning framework that leverages sparsity to make training and inference computationally feasible on CPUs. Our demo repo will help get you familiar with our [BOLT](https://www.thirdai.com/bolt-overview/) API through interactive notebooks.
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+
+
+<!-- GETTING STARTED -->
+# 🚀 Quickstart
+
+This is an example of how you may give instructions on setting up your project locally.
+To get a local copy up and running follow these simple example steps.
+
+<br>
+
+## 🪪 Downloading a License
+
+To use our library, you will first need to register for a free license [here](https://www.thirdai.com/try-bolt/). We will email you a unique download link for your license.
+
+To download your license, run:
+
+```sh
+wget your_download_link ~/license.serialized
+```
+
+This will download the license file to your home directory. Please note: a valid, unexpired license must be placed in your <em>home directory</em>. If a valid license is not found, these demos will fail.
+
+## 💾 Installation
```

Comment:
Is it bad to use h3 instead of h2 for Downloading a License and Installation? Imo it's not visually clear that they're subsections of Quickstart. I know this is usually not recommended, but since the body of text under "Quickstart" is really short maybe it makes sense for this use case?